### PR TITLE
mark default_from_api for empty_dir.medium in Cloud Run API.

### PR DESCRIPTION
**Pull request by bskaplan**
empty_dir.medium for cloud run currently only supports a value of "memory". If it's unset, we use that as the default.

Like with other fields in Cloud Run, we plan to start explicitly capturing the default value and returning it on subsequent gets. Pre-emptively marking the default so Terraform users don't break when the server starts setting this field.

If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [x] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/develop/run-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) before writing my release note below.

```release-note:none

```

### CODE REVIEW

**Review by slevenick — CHANGES_REQUESTED** [review]
Unfortunately adding a default value is a breaking change, so we won't be able to do this unless it's targeted at the major release 5.0.0 coming up. See: https://googlecloudplatform.github.io/magic-modules/develop/make-a-breaking-change/

**Comment by bskaplan** [comment]
It looks like setting default_from_api: true isn't listed as a breaking change. Would it be worth it to set that on the current version?

**Comment by slevenick** [comment]
> It looks like setting default_from_api: true isn't listed as a breaking change. Would it be worth it to set that on the current version?

Yes, that would be a potential option to solve this! I would caution that old versions of the provider that have already been distributed will not handle the new default from the API gracefully, and we often have customers stuck on older versions for a while.

Typically we recommend a soak time for a change like that of 6 months+ if possible before adding the default behavior at the API level

**Comment by bskaplan** [comment]
ok, I switched this to do default_from_api instead. I'll hold off a bit before making the server change but since this field is only in the beta provider, I might do it faster than 6 months.

**Review by slevenick — COMMENTED** [review]
Is this still relevant? It can be merged if so

**Pull request closed without merging**
